bindings/java/src/org/sleuthkit/datamodel/BlackboardArtifact.java

```diff
@@ -169,19 +170,63 @@ public String getDisplayName() {
 	 * @throws TskCoreException if there is a problem creating the description.
 	 */
 	public String getShortDescription() throws TskCoreException {
-		BlackboardAttribute attr = null;
-		if (artifactTypeId == ARTIFACT_TYPE.TSK_WEB_BOOKMARK.getTypeID()) {
-			attr = getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_URL));
+		List<BlackboardAttribute> attrs;
+		attrs = new ArrayList<BlackboardAttribute>();
+		if (artifactTypeId == ARTIFACT_TYPE.TSK_WEB_BOOKMARK.getTypeID()
```

Comment:
You should be able to get the actual ARTIFACT_TYPE enum object corresponding to the Id and switch on that